templates/actor/actor-sheet.html

```diff
@@ -126,7 +126,7 @@ <h2 class="profession"><input name="system.biography.profession" type="text" val
                 {{/if_eq}}
 
                 {{#if_not_eq skill.key 'ritual' }}
-                  <input class="percentile-skill-input" type="text" name="system.skills.{{skill.key}}.proficiency" value="{{skill.proficiency}}" data-dtype="Number" {{#if skill.isCalculatedValue}} disabled {{/if}}/>
+                  <input class="percentile-skill-input {{#if_not_eq skill.proficiency skill.targetProficiency}}target-proficiency-mod{{if_not_eq}}" type="text" name="system.skills.{{skill.key}}.proficiency" value="{{skill.proficiency}}" data-dtype="Number" data-tooltip="{{skill.targetProficiency}}" {{#if skill.isCalculatedValue}} disabled {{/if}}/>
```

Comment:
Missing a closing tag here:

```suggestion
                  <input class="percentile-skill-input {{#if_not_eq skill.proficiency skill.targetProficiency}}target-proficiency-mod{{/if_not_eq}}" type="text" name="system.skills.{{skill.key}}.proficiency" value="{{skill.proficiency}}" data-dtype="Number" data-tooltip="{{skill.targetProficiency}}" {{#if skill.isCalculatedValue}} disabled {{/if}}/>
```